stories/4-Nav.stories.js

```diff
@@ -13,6 +18,9 @@ export const HomePage = () => (
   <Fragment>
```

Comment:
I am seeing the Hero there twice. 

<img width="133" alt="Screen Shot 2021-06-16 at 9 47 57 PM" src="https://user-images.githubusercontent.com/5713670/122333171-973ea180-ceec-11eb-9b97-2c26ab039c0a.png">

Reply:
Removed the extra category for Hero and moved that into Homepage category and will be added upon completion of all the other suggestions.